ocean_lib/ocean/ocean_assets.py

```diff
@@ -281,8 +281,8 @@ def create(
         erc20_publish_market_order_fee_amounts: Optional[List[int]] = None,
         erc20_bytess: Optional[List[List[bytes]]] = None,
         deployed_erc20_tokens: Optional[List[ERC20Token]] = None,
-        encrypt_flag: Optional[bool] = False,
-        compress_flag: Optional[bool] = False,
+        encrypt_flag: Optional[bool] = True,
```

Comment:
make sure in the READMEs you update where we reference ddo encryption defaults. I agree with making the defaults True, but I think we do talk about it in the READMEs, so we need to update that to be in sync.

Reply:
READMEs are updated :white_check_mark: